src/falkirks/simplewarp/command/SimpleWarpCommand.php

```diff
@@ -16,7 +16,7 @@
 use pocketmine\plugin\PluginException;
 use pocketmine\utils\TextFormat;
 
-abstract class SimpleWarpCommand extends Command implements PluginIdentifiableCommand {
+abstract class SimpleWarpCommand extends Command {
```

Comment:
This is a problem. It may be fixed by changing the return type on `getPlugin()` to be `Plugin` so as to match the definition in `PluginIdentifiableCommand`. I can't merge until this is fixed.